src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,155 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    public static final int MIN_USER_AGE = 18;
+    public static final int MIN_PASSWORD_LENGTH = 6;
+    private RegistrationServiceImpl registrationService = new RegistrationServiceImpl();
+    private StorageDaoImpl storageDao = new StorageDaoImpl();
+    private User user;
+
+    @Test
+    void register_sameLogin_NotOk() {
+        User testUser = new User();
+        testUser.setAge(20);
+        testUser.setLogin("Mersedes");
+        testUser.setPassword("king_1975");
+        storageDao.add(testUser);
+        User user = new User();
+        user.setLogin(testUser.getLogin());
+        user.setPassword("somepassword");
+        user.setAge(25);
+        assertThrows(RuntimeException.class, () ->
+                registrationService.register(user).equals(storageDao.get(user.getLogin())));
```

Comment:
```suggestion
        assertThrows(RuntimeException.class, () ->
                registrationService.register(user));
```

We wont get any returnobject from register method if we get exception